package.json

```diff
@@ -120,18 +120,19 @@
     "website:start-preview": "yarn run ts-node ./scripts/website/compile --watch --bundle preview",
     "website:start-testRunner": "yarn run ts-node ./scripts/website/compile --watch --bundle testRunner",
     "website:start-server": "yarn run ts-node ./scripts/website/compile --watch --bundle server",
+    "website:start-server-test": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle server",
     "website:start-static": "cross-env NODE_OPTIONS=\"--max-old-space-size=6144\" TS_NODE_PROJECT=tsconfig/tsconfig.es2017.cjs.json yarn run react-static start",
     "website:start:dev-builds": "rm -rf node_modules/.cache && concurrently -n 'workers,overlay,tools' --kill-others-on-fail \"yarn website:start-workers\" \"yarn website:start-overlay\" \"yarn website:start-tools\"",
     "website:start:dev": "concurrently -n 'preview, testRunner, server, static' --kill-others-on-fail \"yarn website:start-preview\" \"yarn website:start-testRunner\" \"yarn website:start-server\" \"yarn website:start-static\"",
-    "website:start-workers-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --watch --bundle workers",
-    "website:start-overlay-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --watch --bundle overlay",
-    "website:start-tools-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --watch --bundle tools",
-    "website:start-preview-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --watch --bundle preview",
+    "website:start-workers-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle workers",
+    "website:start-overlay-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle overlay",
+    "website:start-tools-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle tools",
+    "website:start-preview-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle preview && yarn run serve dist/preview -p 8080",
     "website:start-testRunner-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle testRunner && yarn run serve dist/testRunner -p 8081",
     "website:start-server-prod": "cross-env NEO_ONE_CACHE=true NEO_ONE_PROD=true yarn run ts-node ./scripts/website/compile --bundle server && yarn run serve dist/server -p 3001",
```

Comment:
@danwbyrne The `website:start-prod` script uses the `website:start-server-test` script instead of the `website:start-server-prod` script. The `website:start-server-prod` script doesn't seem to work. Using the `serve` package just makes the compiled files available. Adding the `--watch` flag like this command does works though. I'm not sure what's the `serve` package is doing or if this is the correct way to handle this. All the other `website:start-prod` scripts use the `serve` package successfully it seems.